src/Audio.md

```diff
@@ -36,7 +36,7 @@ The speaker merges back the two channels, losing the stereo aspect entirely.
 
 The Game Boy's sound chip is called the <abbr title="Audio Processing Unit">APU</abbr>.
 
-The APU runs off the same master clock as the rest of the Game Boy, which is to say, it is fully synced with the CPU and [PPU](<#Rendering Overview>).
+The APU runs off the same master clock as the rest of the Game Boy, which is to say, it is fully synced with the CPU and [PPU](<#Graphics Overview>).
```

Comment:
Actually, "Rendering Overview" is correct, as that is the page that defines the term "PPU".

Reply:
Okay, let's switch back to Rendering then?